src/main/java/com/ait/lienzo/client/core/shape/wires/handlers/impl/WiresConnectorHandlerImpl.java

```diff
@@ -102,21 +116,35 @@ public void onNodeDragEnd(final NodeDragEndEvent event) {
 
     @Override
     public void onNodeMouseClick(final NodeMouseClickEvent event) {
-        if (clickTimer.isRunning()) {
-            clickTimer.cancel();
+        checkRunningTimer(clickTimer);
```

Comment:
I guess there is no need for checking/stopping the scheduled tasks (timers), I would just call the `Timer.cancel()` method